bin/initoverlayfs-install

```diff
@@ -59,30 +83,42 @@ exec_ext4() {
 }
 
 detect_initramfs() {
-  mkdir -p "${INITRAMFS_DUMP_DIR}"
-
-  echo "Extracting initrd into initoverlayfs..."
-
-  file_path="${INITRAMFS_DIR}/initramfs-$kver.img"
-  skipcpio="/usr/lib/dracut/skipcpio"
-  if $skipcpio $file_path | gzip -t - >/dev/null 2>&1; then
-    CAT="zcat"
-  elif $skipcpio $file_path | zstd -q -c -t - >/dev/null 2>&1; then
-    CAT="zstd"
-  elif $skipcpio $file_path | xzcat -t - >/dev/null 2>&1; then
-    CAT="xzcat"
-  elif $skipcpio $file_path | lz4cat -t - >/dev/null 2>&1; then
-    CAT="lz4cat"
-  elif $skipcpio $file_path | bzip2 -t - >/dev/null 2>&1; then
-    CAT="bzcat"
-  elif $skipcpio $file_path | lzop -t - >/dev/null 2>&1; then
-    CAT="lzop"
-  else
-    CAT="cat"
-  fi
+    mkdir -p "${INITRAMFS_DUMP_DIR}"
+
+    echo "Extracting initrd into initoverlayfs..."
 
-  echo "  - File path: ${file_path}"
-  echo "  - Decompressor: $CAT"
+    file_path="${INITRAMFS_DIR}/initramfs-$kver.img"
```

Comment:
might be indentation too?

Reply:
This all has to be reverted in fact `detect_initramfs` should not change